server/issue.go

```diff
@@ -602,8 +602,60 @@ func (s *Server) getPipelineFromIssue(ctx context.Context, issueCreate *api.Issu
 				},
 			},
 		}, nil
+	case api.IssueDatabaseSchemaUpdate:
+		m := api.UpdateSchemaContext{}
+		if err := json.Unmarshal([]byte(issueCreate.CreateContext), &m); err != nil {
+			return nil, err
+		}
+		pc := &api.PipelineCreate{}
+		switch m.MigrationType {
+		case db.Baseline:
+			pc.Name = "Establish database baseline pipeline"
+		case db.Migrate:
+			pc.Name = "Update database schema pipeline"
+		default:
+			return nil, echo.NewHTTPError(http.StatusBadRequest, fmt.Sprintf("Invalid migration type %q", m.MigrationType))
+		}
+
+		for _, d := range m.UpdateSchemaDetailList {
+			if m.MigrationType == db.Migrate && d.Statement == "" {
+				return nil, echo.NewHTTPError(http.StatusBadRequest, "Failed to create issue, sql statement missing")
+			}
+			databaseFind := &api.DatabaseFind{
+				ID: &d.DatabaseID,
+			}
+			database, err := s.composeDatabaseByFind(ctx, databaseFind)
+			if err != nil {
+				if common.ErrorCode(err) == common.NotFound {
+					return nil, echo.NewHTTPError(http.StatusNotFound, fmt.Sprintf("Database ID not found: %d", d.DatabaseID))
+				}
+				return nil, echo.NewHTTPError(http.StatusInternalServerError, fmt.Sprintf("Failed to fetch database ID: %v", d.DatabaseID)).SetInternal(err)
+			}
+
+			taskName := fmt.Sprintf("Establish %q baseline", database.Name)
```

Comment:
So the side effect of doing it in backend is we need to do i18n in backend as well. Ok for now, but we need to figure out a plan later.

Reply:
task name is exposed on the Issue flowbar as well as on the activity panel